### DISCUSSION TITLE
Audible App is discontinued?

### DISCUSSION THREAD

**Discussion by falisca** [audiamus/AaxAudioConverter#99] [Q&A] [answered]
Hi, a week ago I opened the Audible app, but there was an orrible surprise.
I can't make the app to refresh the library, I tryed a manual refresh, reinstallation ecc...
After ne installation my library on the app is empty.
Then i contacted Audible support and they sayd the Windows 10 app is no more developed and it will not work anymore!
Is that confirmed?
I' was using the audible.it marketplace, that like the audible.co.jp was storically different from other marketplaces.
I tryed on two different Windows 10 machines, with the same result: Audible app is activated fine, but empty library.

**Comment by Sailfish** [comment, marked as answer]
My Win10 Audible (v10.5.67.0) app refreshes fine. Could it be that you need to re-register your Win10 Audible app?

 

From: Falisca ***@***.*** 
Sent: Thursday, November 18, 2021 9:14 PM
To: audiamus/AaxAudioConverter
Cc: Subscribed
Subject: [audiamus/AaxAudioConverter] Audible App is discontinued? (Discussion #99)

 

Hi, a week ago I opened the Audible app, but there was an orrible surprise.
I can't make the app to refresh the library, I tryed a manual refresh, reinstallation ecc...
After ne installation my library on the app is empty.
Then i contacted Audible support and they sayd the Windows 10 app is no more developed and it will not work anymore!
Is that confirmed?
I' was using the audible.it marketplace, that like the audible.co.jp was storically different from other marketplaces.
I tryed on two different Windows 10 machines, with the same result: Audible app is activated fine, but empty library.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub <https://github.com/audiamus/AaxAudioConverter/discussions/99> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAA23YWFRIMCJ5VJVWJH5FDUMXMJ5ANCNFSM5ILJOF3A> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .   <https://github.com/notifications/beacon/AAA23YVPZ2MT4I6L6DOYKN3UMXMJ5A5CNFSM5ILJOF3KYY3PNVWWK3TUL52HS4DFVJCGS43DOVZXG2LPN2VGG33NNVSW45C7NFSM4ABYLZGQ.gif>

**Reply by falisca** [reply]
Hi, sorry if i don't understand, what you mean with re-register? re-registering is a procedure made on the audible account, Windows registry or something i don't nowk yet? I've only reinstalled from Microsoft store a couple of times.
In any case the response from Audible support seems weyrd to me.

**Reply by audiamus** [reply]
If I remember correctly, you have to log-in to the "region" where you have your Audible account. I think, by default the Audible App would take you to the US region.
